cmd/gateway/main.go

```diff
@@ -28,16 +30,23 @@ func main() {
 	rc := ws.NewReceptorController(cm, wsMux, kw)
 	rc.Routes()
 
-	mgmtMux := mux.NewRouter()
-	mgmtServer := c.NewManagementServer(cm, mgmtMux)
+	apiMux := mux.NewRouter()
+
+	apiSpecServer := api.NewApiSpecServer(apiMux)
+	apiSpecServer.Routes()
+
+	securedApiMux := apiMux.PathPrefix("/").Subrouter()
+	securedApiMux.Use(identity.EnforceIdentity)
```

Comment:
We could probably do the same with wsMux above

Reply:
Good point!  :+1:   I'll do that in another PR.